src/plugins/discover/public/application/angular/doc_table/doc_table.html

```diff
@@ -87,7 +87,7 @@
       on-remove-column="onRemoveColumn"
     ></thead>
     <tbody>
-      <tr ng-repeat="row in hits|limitTo:limit track by row._index+row._type+row._id+row._score+row._version+row._routing"
+      <tr ng-repeat="row in hits|limitTo:limit track by row._index+row._type+row._id+row._score+row._version+row._routing+hideUnmapped"
```

Comment:
I added a functional test for this because it was super weird to me. The cell contents don't update unless one of these properties is updated.

Reply:
yes, this seems to be the concatenated variable Angular used to track updates, great that you've fixed it and even greater you've added a functional test for it 👍